modules/apps/foundation/login/login-web/src/main/resources/META-INF/resources/forgot_password.jsp

```diff
@@ -167,4 +170,64 @@ if (reminderAttempts == null) {
 	</aui:fieldset>
 </aui:form>
 
-<liferay-util:include page="/navigation.jsp" servletContext="<%= application %>" />
\ No newline at end of file
+<aui:script>
+	var dialog = Liferay.Util.getWindow();
```

Comment:
Also, since so much of this logic is duplicated, can we either place the code that's the same in a .jspf, or move it to a function in a JS file?